CHANGELOG.rst

```diff
@@ -15,6 +15,10 @@ Unreleased
 ~~~~~~~~~~
 *
 
+[0.5.3] - 2020-12-08
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Celery multiple versions testing.
```

Comment:
I don't think we technically need a version bump for this; none of the changes in this PR seem to impact the package uploaded to PyPI.  Might be worth it if we specifically mention here that we added testing for celery 5, I suppose.

Reply:
revert the releasing.